nectar/downloaders/threaded.py

```diff
@@ -200,9 +208,19 @@ def _fetch(self, request, session):
             if netloc in self.failed_netlocs:
                 raise SkipLocation()
 
-            response = session.get(
-                request.url, headers=headers, timeout=(self.config.connect_timeout, self.config.read_timeout)
-            )
+            for attempt in xrange(self.tries):
+                try:
```

Comment:
I would log here, if ```attempt > 0```, that a request is being retried. Combined with the log statement you'll add below, the user will be able to piece together that a request failed, and whether or not the retry succeeded.